src/space-accessors.js

```diff
@@ -20,11 +20,6 @@ hooks.add("colorspace-init-end", space => {
 });
 
 function addSpaceAccessors (id, space) {
-	// Coordinates can be looked up by both id and name
-	let coordIds = Object.keys(space.coords);
-	let coordNames = Object.values(space.coords).map(c => c.name);
-
-
```

Comment:
Wow, these were entirely unused?!

Reply:
They were made obsolete by https://github.com/color-js/color.js/commit/cd0b54311fd6f5c80545aaa45022c99192f1b246, but not removed.